api/organisations/tasks.py

```diff
@@ -73,3 +86,93 @@ def finish_subscription_cancellation():
     ):
         subscription.organisation.cancel_users()
         subscription.save_as_free_subscription()
+
+
+def send_admin_api_usage_notification(
+    organisation: Organisation, matched_threshold: int
+) -> None:
+    """
+    Send notification to admins that the API has breached a threshold.
+    """
+    recipient_list = list(
+        FFAdminUser.objects.filter(
+            userorganisation__organisation=organisation,
+            userorganisation__role=OrganisationRole.ADMIN,
+        ).values_list("email", flat=True)
+    )
```

Comment:
One thing that came up in a conversation with Greg about this yesterday is whether we can change the distribution list based on the severity of the threshold. Essentially, for the 100 / 120 thresholds, we want to email everyone in the organisation.

Reply:
Ok that's no problem.

Reply:
This has been added.